### PR TITLE
docs(common): copy spaces styles on click

### LINKED ISSUE #197: [FEATURE] Copy to clipboard for spaces styles by click
# 🚀 Feature request

### Is your feature request related to a problem?

I'm always frustrated when I need to copy some class names by hand instead of clicking them and getting the selected class names in the clipboard.

image

### Describe the solution you'd like

I want to be able to copy spaces class names to the clipboard by click. Like in the typography section.

### PR DESCRIPTION
**Pull request by fynnfeldpausch**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [x] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Other... Please describe:

## What is the current behavior?

Closes #197

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

File: projects/demo/src/modules/markup/spaces/examples/1/index.html [hunk 1]
```diff
@@ -1,59 +1,137 @@
 <h2 i18n class="title">Margin top <code>tui-space_top-&lt;value&gt;</code></h2>
 <div class="row">
-    <div class="example tui-space_top-1 tui-space_right-2">tui-space_top-1</div>
-    <div class="example tui-space_top-2 tui-space_right-2">tui-space_top-2</div>
-    <div class="example tui-space_top-3 tui-space_right-2">tui-space_top-3</div>
-    <div class="example tui-space_top-4 tui-space_right-2">tui-space_top-4</div>
-    <div class="example tui-space_top-5 tui-space_right-2">tui-space_top-5</div>
+    <div class="example tui-space_top-1 tui-space_right-2">
+        <tui-doc-copy [cdkCopyToClipboard]="'tui-space_top-1'">
```
Comment (MarsiBarsi) [inline projects/demo/src/modules/markup/spaces/examples/1/index.html:4]: Hey 👋
Unfortunately, there is no linter rules or public code style guideline for this (we'll think about it), but we prefer to write `cdkCopyToClipboard="tui-space_top-1"` passing a string

Comment (fynnfeldpausch) [reply projects/demo/src/modules/markup/spaces/examples/1/index.html:4]: Haha I prefer it as well and had written it before but changed it to match the template of the typography demo 🙈

Comment (fynnfeldpausch) [reply projects/demo/src/modules/markup/spaces/examples/1/index.html]: @MarsiBarsi fixed for spaces and typo docs 👍

**Review by MarsiBarsi — APPROVED** [review]

File: projects/demo/src/modules/markup/spaces/examples/1/index.html (hunk 1 above)
Comment (MarsiBarsi) [reply projects/demo/src/modules/markup/spaces/examples/1/index.html]: oh, it seems to be a mistake by us, thank you

File: projects/demo/src/modules/markup/spaces/spaces.module.ts [hunk 2]
```diff
@@ -1,14 +1,17 @@
+import { ClipboardModule } from '@angular/cdk/clipboard';
```
Comment (waterplea) [inline projects/demo/src/modules/markup/spaces/spaces.module.ts:1]: That's weird. Prettier should have had removed the spaces on precommit 🤔

Comment (MarsiBarsi) [reply projects/demo/src/modules/markup/spaces/spaces.module.ts:1]: @fynnfeldpausch what IDE do you use?

Comment (fynnfeldpausch) [reply projects/demo/src/modules/markup/spaces/spaces.module.ts:1]: VSC 🤔

Comment (MarsiBarsi) [reply projects/demo/src/modules/markup/spaces/spaces.module.ts:1]: Did you need to delay something like 1-2 seconds making a commit or was that instantly?

**Pull request merged**